Ryujinx.HLE/HOS/Services/Hid/IHidServer.cs

```diff
@@ -688,7 +706,7 @@ public ResultCode GetNpadJoyHoldType(ServiceCtx context)
         // SetNpadJoyAssignmentModeSingleByDefault(uint HidControllerId, nn::applet::AppletResourceUserId)
         public ResultCode SetNpadJoyAssignmentModeSingleByDefault(ServiceCtx context)
         {
-            ControllerId hidControllerId      = (ControllerId)context.RequestData.ReadInt32();
+            PlayerIndex  hidControllerId      = (PlayerIndex)context.RequestData.ReadInt32();
```

Comment:
```suggestion
            PlayerIndex hidControllerId      = (PlayerIndex)context.RequestData.ReadInt32();
```
Don’t forget to remove the space in the line above too